hasura/migrations/default/1648096542742_alter_table_public_profiles_add_check_constraint_valid_website/up.sql

```diff
@@ -0,0 +1,8 @@
+update "public"."profiles" set website = CONCAT('https://',website) where website not ilike 'https://%' and website not ilike 'http://%' and website is not null;
+-- update these three specific users, making sure they have these y.at addresses
+update "public"."profiles" set website = RTRIM(website) where website is not null;
```

Comment:
hmm why not use `TRIM` instead

Reply:
because the only example I found was right trim 🤷‍♀️